.circleci/config.yml

```diff
@@ -9,6 +9,7 @@ defaults: &defaults
 version: 2.1
 orbs:
   node: circleci/node@5.1.0
+  publish-docs: infinitered/publish-docs@0.4.8
```

Comment:
I think this can be bumped to 0.4.9

https://circleci.com/developer/orbs/orb/infinitered/publish-docs

Reply:
Done!